sporks.json

```diff
@@ -1309,6 +1309,45 @@
       }
     },
     "testnet": {
+      "testnet48": {
+        "id": 48,
+        "live": true,
+        "name": "Testnet48",
+        "sporkTime": "2023-10-25T14:30:00Z",
```

Comment:
Spork time on root snapshot is `2023-10-25T15:21:04Z`

https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/root-protocol-state-snapshot.json